samples/caf_sensor_manager/README.rst

```diff
@@ -31,6 +32,24 @@ This module handles the data received from the :ref:`caf_sensor_data_aggregator`
 If you are running this sample on an SoC with multiple cores, the workload simulator module (``workload_sim``) is placed on the second core.
 All communication between the cores is done using :ref:`event_manager_proxy` and Zephyr subsystem :file:`include/ipc/ipc_service.h`.
 
+Configuration
+*************
+
+|config|
+
+Single core configuration
+=========================
+
+For the MCU with multiple cores, the default configuration will use one core to simulate the sensor and the other core to process the sensor.
+These multiple core MCU can support single core configuration, where the sensor is simulated and processed on the single, selected core.
+The configuration is placed in the :file:`boards/<board>_singlecore.conf file`.
```

Comment:
```suggestion
The configuration is placed in the :file:`boards/<board>_singlecore.conf` file.
```
not sure if the 'file' after .conf was supposed to be kept as a part of the path